src/sigal/image.py

```diff
@@ -220,7 +225,11 @@ def get_exif_data(filename):
 
     try:
         with warnings.catch_warnings(record=True) as caught_warnings:
-            exif = img._getexif() or {}
+            exif = {}
+            if hasattr(img, "_getexif") and callable(img._getexif):
+                exif = img._getexif()
+            elif "exif" in img.info:
+                exif = img.getexif()._get_merged_dict()
```

Comment:
`img._getexif()` is an alias for `img.getexif()._get_merged_dict()` so we can probably switch to the public methods, which I think are supported since a long time ? (Our min version of pillow is 8.0 currently)

Reply:
Gladly!
I just wasn't sure if I would break any weird combination, because both variants use some private API.
But let tox find out :-)